config/sync/os2loop.settings.yml

```diff
@@ -1,11 +1,11 @@
 node_type:
   os2loop_documents_collection: os2loop_documents_collection
   os2loop_documents_document: os2loop_documents_document
+  os2loop_post: os2loop_post
```

Comment:
This change is not related to media and should not be a part of this pull request.